woocommerce/woocommerce
# [Experimental] Product Filters Redesign > Overlay: Replace Product Filters Block with Template Part

### Changes proposed in this Pull Request:





Closes #49998 .

In this PR, the Product Filters Block within the Product Filter Overlay template part are being replaced by the Product Filters template part. The reason for this change is that we want to keep the filters in sync with the template part.



### How to test the changes in this Pull Request:



> [!NOTE]  
> **Pre-requisite**
> - If you're using WC Beta Tester plugin, make sure the `experimental-blocks` feature is enabled (Tools > WCA Test Helper > Features).

1. Navigate to Product Filters Overlay template (Appearance > Editor > Patterns > General > Filters Overlay (Experimental).
2. Click on it to open the WordPress editor.
3. In the editor, open the Blocks List View.
4. Make sure you can see the Navigation block followed by the Product Filters template part.




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment  Replace the Product Filters block within the Product Filters Overlay template part with the Product Filters template part. This functionality is experimental and not yet available.

</details>
